packages/fxa-auth-server/test/lib/util.js

```diff
@@ -36,7 +37,22 @@ function decodeJWT(b64) {
   };
 }
 
+function assertSecurityHeaders(res, expect = {}) {
```

Comment:
copied from `fxa-oauth-server/test/lib/util.js` minus the bit about cache-control